talk/morelanguage/lambda.tex

```diff
@@ -344,6 +356,37 @@
   \end{alertblock}
 \end{frame}
 
+\begin{frame}[fragile]
+  \frametitlecpp[17]{lambdas and the visitor pattern}
```

Comment:
I wonder whether the slide should be moved to the ones about `std::variant`.

Reply:
Well, at that stage we did not yet cover lambdas, so it would be weird.

Reply:
issue #246 was open concerning a possible refactoring of the slides there. And we've decided to merge this as is for the moment